Makefile

```diff
@@ -136,6 +143,15 @@ ifeq ("$(CC_IS_EMCC)", "1")
 CFLAGS += -mtail-call
 endif
 
+# Build emscripten-port SDL
+ifeq ("$(CC_IS_EMCC)", "1")
+ifeq ($(call has, SDL), 1)
+EMCC_PORT_FLAGS := -sUSE_SDL=2 -sSDL2_MIXER_FORMATS=wav,mid -sUSE_SDL_MIXER=2
```

Comment:
Simply name it `CFLAGS_emcc`.

Reply:
> Simply name it `CFLAGS_emcc_port`.

No more need  `CFLAGS_emcc_port` since we can simply concatenate flags to `CFLAGS_emcc`.